test/ibmq/test_ibmq_job_attributes.py

```diff
@@ -20,8 +20,9 @@
 from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister
 from qiskit.providers import JobStatus
 from qiskit.providers.ibmq import least_busy
-from qiskit.providers.ibmq.job.exceptions import IBMQJobFailureError, JobError
+from qiskit.providers.ibmq.job.exceptions import (IBMQJobFailureError, JobError)
```

Comment:
Unintended change?

Reply:
Yeah, I'm not sure why I changed that, thanks for the catch @diego-plan9! 😅